pkg/testing/integration/program.go

```diff
@@ -962,6 +962,17 @@ func (pt *ProgramTester) runCommand(name string, args []string, wd string) error
 	return RunCommand(pt.t, name, args, wd, pt.opts)
 }
 
+// RunPulumiCommand runs a Pulumi command in the project directory.
+// For example:
+//
+//	pt.RunPulumiCommand("preview", "--stack", "dev")
+func (pt *ProgramTester) RunPulumiCommand(name string, args ...string) error {
+	// RunPulumiCommand uses 'name' for logging only.
```

Comment:
I was initially confused by this comment. I think you mean:

```suggestion
	// pt.runPulumiCommand uses 'name' for logging only.
```

Reply:
Oops, yes.